boards/beagle/beagleconnect_freedom/Kconfig.beagleconnect_freedom

```diff
@@ -6,3 +6,4 @@
 
 config BOARD_BEAGLECONNECT_FREEDOM
 	select SOC_CC1352P7
+	select SKY13317
```

Comment:
Why do you need to do this if the default is y with depends on DT_ENABLED? and you it enabled in board DT also.